### DISCUSSION TITLE
Add ability to track views/ collection usage

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5763] [Feature Requests]
Specify should support some form of embeded view tracking for records and URLs in general to gain insight into usage of the collection data and popularity by record (especially when anonymous access exists)

**Comment by maxpatiiuk** [comment]
Is this same as #2848, except #2848 is for physical visitors and this is for digital visitors?

**Comment by grantfitzsimmons** [comment]
This request is for an automated process that tracks collections usage in Specify, potentially using the stats system we engineer or use in the future for 7 and allow the collection manager to see those (views per record, traffic by URL)

#2848 is for physical visitors as you said
